frontend/src/Components/pages/Settings.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import ROUTES from "../Routes/ROUTES";
+import CenteredContainer from "../ReusableComponents/CenteredContainer";
+import Label from "../ReusableComponents/Label";
+
+function Settings() {
+   return (
+      <CenteredContainer>
+         <form>
+            <h2 className="text-center">SKY-NEST</h2>
+            <p className="mb-4 p-0 text-center text-secondary">Edit your account</p>
+            <div className="row">
+               <div className="col-md-6">
+                  <div className="form-outline mb-1">
```

Comment:
I would maybe extract this kind of divs into component, I see it is repeating in this component.